Triv fix arm kconfig

laodzu opened the pull request:
These trivial Kconfig patches align upper and lower case in the SoC selection in menuconfig.

Before:

> ( ) ARM Beetle MCU Series
> ( ) ARM DesignStart SoC Series
> ( ) ARM MPS2 MCU Series
> ( ) ARM v2m MUSCA MCU Series
> ( ) arm v2m MUSCA B1 MCU Series
> ( ) arm v2m MUSCA-S1 MCU Series

and 

> ( ) STM32F0x Series MCU
> ( ) STM32F1x Series MCU
> ( ) stm32f2x Series MCU
> ( ) STM32F3x Series MCU
> ( ) STM32F4x Series MCU
> ( ) STM32F7x Series MCU
> ( ) STM32G0x Series MCU
> ( ) STM32G4x Series MCU
> ( ) STM32H7x Series MCU
> ( ) STM32L0x Series MCU
> ( ) STM32L1x Series MCU
> ( ) STM32L4x Series MCU
> ( ) STM32L5x Series MCU
> ( ) STM32MP15 Series MPU
> ( ) STM32WBx Series MCU

After the changes:

> ( ) Arm Beetle MCU Series
> ( ) Arm DesignStart SoC Series
> ( ) Arm MPS2 MCU Series
> ( ) Arm v2m MUSCA MCU Series
> ( ) Arm v2m MUSCA B1 MCU Series
> ( ) Arm v2m MUSCA-S1 MCU Series

> ( ) STM32F0x Series MCU
> ( ) STM32F1x Series MCU
> ( ) STM32F2x Series MCU
> ( ) STM32F3x Series MCU
> ( ) STM32F4x Series MCU
> ( ) STM32F7x Series MCU
> ( ) STM32G0x Series MCU
> ( ) STM32G4x Series MCU
> ( ) STM32H7x Series MCU
> ( ) STM32L0x Series MCU
> ( ) STM32L1x Series MCU
> ( ) STM32L4x Series MCU
> ( ) STM32L5x Series MCU
> ( ) STM32MP15 Series MPU
> ( ) STM32WBx Series MCU

laodzu committed:
soc: arm: st_stm32f2x: Convert Kconfig input prompt to uppercase

The STM32F2x was the only one of the STM32 series with a lowercase
input prompt.  This looks odd in the SoC selection in 'menuconfig'.

Signed-off-by: Detlev Zundel <dzu@member.fsf.org>

laodzu committed:
soc: arm: arm: Align Kconfig input prompt for Arm SoCs

According to https://www.arm.com the correct spelling is now "Arm".
Change the Kconfig input prompts to comply.

Signed-off-by: Detlev Zundel <dzu@member.fsf.org>

ioannisg commented on soc/arm/arm/musca_b1/Kconfig.series:
```diff
@@ -2,7 +2,7 @@
 
 config SOC_SERIES_MUSCA_B1
-	bool "arm v2m MUSCA B1 MCU Series"
+	bool "ARM v2m MUSCA B1 MCU Series"
```
Has to be Arm, then, to comply with the newest standard.

laodzu replied:
Thanks for this information.  Do you have an URL handy that I can put in the commit msg?

So this will require changing the other Arm SoCs as well, so I'll update the PR accordingly.

ioannisg replied:
https://www.arm.com/

ioannisg commented on soc/arm/arm/musca_s1/Kconfig.series:
```diff
@@ -2,7 +2,7 @@
 
 config SOC_SERIES_MUSCA_S1
-	bool "arm v2m MUSCA-S1 MCU Series"
+	bool "ARM v2m MUSCA-S1 MCU Series"
```
ditto

laodzu committed:
soc: arm: designstart: Fix Kconfig LEADING_SPACE checkpatch problem

This is a trivial patch replacing the spaces at the beginning of the
line with tabs.  Otherwise checkpatch will not be happy with any
changes in here.

Signed-off-by: Detlev Zundel <dzu@member.fsf.org>
